Resources/es.json

```diff
@@ -2648,6 +2670,129 @@
+			"code": "Código de reciclaje"
+		},
+		"views": {
+			"edit": {
+				"title": "Editar hechos"
+			},
+			"create": {
+				"title": "Crear nuevo hecho"
+			}
+		},
+		"dialogs": {
+			"delete": "¿Realmente deseas eliminar esta información alimentaria?",
+			"calculate": {
+				"nutritionalInformation": "<strong>ATENCIÓN</strong> La información nutricional ahora se calcula en base a los datos analíticos existentes de alcohol, azúcar residual y acidez total. Además, se incluye el valor de la glicerina si se especifica. ¿Quieres calcular la información nutricional ahora?",
+				"ingredients": "Los ingredientes ahora se calculan de forma estandarizada. <strong>ATENCIÓN, los alérgenos no vuelven a aparecer por separado.</strong> ¿Quieres identificar los ingredientes ahora?"
+			}
+		},
+		"toasts": {
+			"created": "Información alimentaria creada",
+			"changed": "La información alimentaria cambió",
+			"deleted": "Información alimentaria eliminada",
+			"calculated": "Información alimentaria determinada automáticamente"
+		},
+		"types": {
+			"aluminium": "Aluminio",
+			"antioxidants": "Antioxidantes",
+			"brownGlas": "Vidrio marrón",
+			"carbohydrates": "carbohidratos",
+			"clearglas": "Vaso transparente",
+			"cork": "Corcho",
+			"corrugatedCardboard": "Cartón corrugado",
+			"cotton": "Algodón",
+			"energy": "Energía",
+			"fat": "Gordo",
+			"glasAluminium": "Vidrio/Aluminio",
+			"glasMetal": "Vidrio/Varios metales",
+			"glasPlastic": "Vidrio/Plástico",
+			"glasTinplate": "vidrio/hojalata",
+			"grapes": "Uvas",
+			"greenGlas": "Vidrio verde",
+			"iron": "Hierro",
+			"jute": "Yute",
+			"otherCardboard": "Otros cartones",
+			"paper": "Papel",
+			"paperAluminium": "Papel y cartón/aluminio",
+			"paperMetal": "Papel y cartón/metales varios",
+			"paperPlastic": "Papel y cartón/plástico",
+			"paperPlasticAlu": "Papel y cartón/aluminio",
+			"paperPlasticMetal": "Papel y cartón/plástico/aluminio/hojalata",
+			"paperTinplate": "Papel y cartón/hojalata",
+			"pet": "Tereftalato de polietileno - PET",
+			"petHD": "Polietileno de alta densidad",
+			"petLD": "Polietileno de baja densidad",
+			"plasticAluminium": "Plástico/Aluminio",
+			"plasticMetal": "Plástico/Metales varios",
+			"plasticTinplate": "Plástico/Hojalata",
+			"pp": "Polipropileno",
+			"preservatives": "Conservantes",
+			"protein": "Proteína",
+			"ps": "Poliestireno",
+			"pvc": "Cloruro de polivinilo",
+			"salt": "Sal",
+			"saturatedFattyAcids": "Ácidos grasos saturados",
+			"sugar": "Azúcar",
+			"unsaturatedFattyAcids": "Ácidos grasos insaturados",
+			"wood": "Madera"
+		},
+		"factgroups": {
+			"nutritionalInformation": {
+				"label": "Información nutricional",
+				"description": "<strong>Asegúrese de que la información nutricional se proporcione por 100 ml.</strong> La información sobre el contenido de azúcar y energía se calcula automáticamente a partir de los datos del vino: contenido de alcohol, dulzor residual y acidez.</strong>",
```

Comment:
Hier auch wieder schließendes Strong zu viel. Wahrscheinlich auch in allen weiteren Sprachen.